Stripe/STPTheme.m

```diff
@@ -60,6 +61,8 @@ - (instancetype)init {
         _errorColor = STPThemeDefaultErrorColor;
         _font = STPThemeDefaultFont;
         _emphasisFont = STPThemeDefaultMediumFont;
+        _translucentNavigationBar = NO;
+        _barStyle = -1;
```

Comment:
Ah I see. Yeah I think this is fine, but maybe put some comments in where you assign it in init to make it clear.